src/libraries/Microsoft.VisualBasic.Core/src/Microsoft/VisualBasic/FileIO/TextFieldParser.vb

```diff
@@ -1463,17 +1463,14 @@ Namespace Microsoft.VisualBasic.FileIO
         ' Regex used with BuildField
         Private m_DelimiterWithEndCharsRegex As Regex
 
-        ' Options used for regular expressions
-        Private Const REGEX_OPTIONS As RegexOptions = RegexOptions.CultureInvariant
-
         ' Codes for whitespace as used by String.Trim excluding line end chars as those are handled separately
         Private m_WhitespaceCodes() As Integer = {&H9, &HB, &HC, &H20, &H85, &HA0, &H1680, &H2000, &H2001, &H2002, &H2003, &H2004, &H2005, &H2006, &H2007, &H2008, &H2009, &H200A, &H200B, &H2028, &H2029, &H3000, &HFEFF}
 
         ' Regular expression used to find beginning quotes ignore spaces and tabs
         Private m_BeginQuotesRegex As Regex
 
         ' Regular expression for whitespace
-        Private m_WhiteSpaceRegEx As Regex = New Regex("\s", REGEX_OPTIONS)
```

Comment:
The results are the same, but without this change it passes the flag through only to ignore it. It's basically dead code in this context.

Reply:
Right, this is just removing dead code that has some overhead.

Reply:
Would it make sense to pass `RegexOptions.None` instead then, or does that also introduce overhead? It seems useful to have a single constant that is used in all calls, so that if the options need to change in future (perhaps unlikely), it'll be easier to track.

Reply:
We _could_, though it's not typical to do that.  The actual execution overhead is minimal; a few checks to see whether the options contains various values.  The biggest downside to passing any RegexOptions when they're not necessary is that the trimmer is then unable to trim out the code that overload depends on, which includes the whole RegexOptions.Compiled and RegexOptions.NonBacktracking implementation.